src/components/ui/Footer.vue

```diff
@@ -37,7 +37,10 @@ export default {
           abbr: 'Community Participation Guidelines',
         },
         {
-          link: this.globalLinks.faq,
+          link:
+            this.scope.isStaff || this.scope.isNdaed
+              ? this.globalLinks.faq
```

Comment:
This is a bit nitpicky, but if this ```faq``` field is the 'private' field, can we just make it be ```faqPrivate``` for clarity sake?

Reply:
Agreed, I left it as just `faq` as that's what it was already called, but that doesn't make much sense to fresh eyes. I think I'll probably call it `faqNda` - as that's the visibility of the post in Discourse.